src/components/lender/active-loans/ParticlesBackground.tsx

```
@@ -0,0 +1,106 @@
+
+export default function ParticlesBackground() {
+  const canvasRef = useRef<HTMLCanvasElement>(null)
+  const particles = useRef<Particle[]>([])
+  const animationFrameId = useRef<number>(0)
+
+  useEffect(() => {
+    const canvas = canvasRef.current
+
+    if (!canvas) return
+
+    const ctx = canvas.getContext("2d")!
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)